docpipeline/generate.py

```diff
@@ -220,9 +220,10 @@ def download_xrefs(client, bucket):
     xrefs_dir.mkdir(parents=True, exist_ok=True)
     xrefs = []
     for xref_blob in client.list_blobs(bucket, prefix=XREFS_DIR_NAME):
-        xref_path = str(pathlib.Path(xref_blob.name).absolute())
+        xref_path = pathlib.Path(xref_blob.name).absolute()
+        xref_path.parents[0].mkdir(parents=True, exist_ok=True)
```

Comment:
Just use `parent` instead of `parents[0]`?

Reply:
Done.